src/vue/common/CollectionLoader.spec.js

```diff
@@ -0,0 +1,114 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import CollectionLoader from './CollectionLoader'
+import { globalize } from '../filters/globalize'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+
+const localVue = createLocalVue()
+localVue.filter('globalize', globalize)
+let wrapper
+const someData = []
+const requestPerPage = 10
+const someEvent = 'some-event'
```

Comment:
I'd store the value in a variable only for the tests I need to assert it. In other cases you just need to pass any random string to the `loadPage` so it's reusability gives you nothing